### DISCUSSION TITLE
how to setup this project for desktop application ??

### DISCUSSION THREAD

**Discussion by milansoliya4210** [WorldWindEarth/WorldWindKotlin#62] [Q&A] [answered]
How can I use this project for multi platform or Desktop Application ?

**Comment by EMaksymenko** [comment, marked as answer]
Hi. Currently, only Web and Android platforms are ready to use. Pure Java (e.g. JavaFX) usage based on JOGL or LWJGL is incomplete yet.

So the only option of desctop usage for now is web based.

**Comment by milansoliya4210** [comment]
Hi @EMaksymenko this project can support Kotlin Multiplatform Development as of now ?

**Reply by EMaksymenko** [reply]
Yes, but only for Android and Web. No IOS support, and it is not planned.
